HybridCamLib/src/cam/CamView.swift

```diff
@@ -16,6 +16,10 @@ open class CamView: UIView {
    public let photoOutput: AVCapturePhotoOutput = .init()
    /*State*/
    public var flashMode: AVCaptureDevice.FlashMode = .off
+   
+   //public var currentCamera blurtime: needed, right?
+   public var startingX: CGFloat = 0
```

Comment:
it's a good practice to spell things out according to @sindresorhus 😏 what about: longPressStartPos: CGPoint. We can maybe use a point and then use start.distanceTo(end) to measure the diagonal distance not just x or y. This makes the question of flipped coordinate system irrelevant as well.

Reply:
Sounds like an awesome idea! 😊I would just move this long-press gesture away from the preview to the already existing long-press gesture of the record button. And if you do it I suggest adding some boundaries (like the borders of the button or even a few pixels away from the button's borders) before it starts zooming in.

Reply:
yeah. only zoom when distance is minMax(60,400) etc